news/l_progress.rst

```diff
@@ -0,0 +1,11 @@
+**Added:** added a helper to list progress by person
```

Comment:
this needs to be one line below with an asterisk in front.  i.e., 
```
**Added**
 * a helper that lists progress by person from their projecta
```

Reply:
Done.